proto/babylon/zoneconcierge/v1/query.proto

```diff
@@ -63,3 +81,35 @@ message FinalizedBSNData {
   // Proofs of finality
   ProofFinalizedHeader proof = 6;
 }
+
+// QueryLatestEpochHeadersRequest is request type for the
+// QueryLatestEpochHeadersRequest RPC method.
+message QueryLatestEpochHeaderRequest {
+  string consumer_id = 1;
+}
+
+// QueryLatestEpochHeaderResponse is response type for the
+// QueryLatestEpochHeaderResponse RPC method.
+message QueryLatestEpochHeaderResponse {
+  IndexedHeader header = 1;
+}
```

Comment:
Since we are distinguishing DB objects and query API objects, we will need to avoid using `IndexedHeader`, `BTCChainSegment`, and `ProofEpochSealed`, and instead create new response structs e.g., `IndexedHeaderResponse`. At this stage they can be exactly same as the DB objects, and they might diverge in the futrue